ES/ESmy019.xml

```diff
@@ -31,63 +30,154 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         <condition key="deficient"/>
                      </supportDesc>
+                     </objectDesc>
+
+                     <bindingDesc>
+                        <binding contemporary="true">
+                           <decoNote xml:id="b1">Two wooden boards, crudely cut. Four sewing
+                              stations.</decoNote>
+                           <decoNote xml:id="b2" type="bindingMaterial">
+                              <material key="wood"/>
+                           </decoNote>
+                           </binding>
+                     </bindingDesc>
+                     </physDesc>
+
+                     <history>
+                        <origin>
+                           <origDate notBefore="1841" notAfter="1867">1841-67</origDate> for
+                           <ref target="#p1"/>, and <origDate notBefore="1820" notAfter="1867"> the same or
+                         slightly earlier time </origDate> for <ref target="#p2"/>.
+                         At an unknown time, <ref target="#p2">Unit 2</ref> was included into the binding of
+                        <ref target="#p1">Unit 1</ref>.
+                        </origin>
+                     </history>
+
+                     <msPart xml:id="p1">
+                        <msIdentifier/>
+                        <msContents>
+                           <summary/>
+
+                           <msItem xml:id="p1_i1">
+                              <locus from="2ra" to="17va"/>
+                              <title type="complete" ref="LIT6166MiraclesJohnBaptist">Ten
+                                miracles of John the Baptist</title>
+                              <incipit xml:lang="gez"/>
+                              <note>
+                                  Cp. Mss Gunda Gunde GG-024, RQM-029.
+                              </note>
+                              </msItem>
+
+                              <msItem xml:id="p1_i2">
+                                 <locus from="17vb" to="19rb"></locus>
+                                 <title type="complete" ref="LIT4717TaammerLi">Two miracles of Libānos</title>
+                                 <incipit xml:lang="gez"/>
+                                 <explicit xml:lang="gez"></explicit>
+                                 <note>
+                                   For the edition, s. <bibl><ptr target="bm:Bausi2003Libanos"/>
+                                   <citedRange unit="paragraph">282-283</citedRange></bibl>, and
+                                   <bibl><ptr target="bm:Bausi2003Libanos"/>
+                                   <citedRange unit="paragraph">290-291</citedRange></bibl>.
+                                   The red ink of  is faded and poorly readable.
+                                   The subscription (<locus from="18vb" to="19ra"/>) states
+                                   that the manuscript was purchased and donated to the church of Dabra
```

Comment:
you need the subscription as an extra element
`<explicit type="subscriptio"><locus from="18vb" to="19ra"/><note>The subscription states...</note></explicit>`

Reply:
Not sure I understood it right.